Decisions.Microsoft365.Exchange/Decisions.Microsoft365.Exchange.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="DecisionsSDK" Version="8.19.0" />
+    <PackageReference Include="DecisionsSDK" Version="9.0.0" />
```

Comment:
Please update to latest SDK (9.1.0 - published Friday)

Reply:
Updated